external/model-preparation-algorithm/mpa_tasks/apis/classification/task.py

```diff
@@ -313,13 +318,13 @@ def _init_recipe(self):
         train_type = self._hyperparams.algo_backend.train_type
         logger.info(f"train type = {train_type}")
 
-        recipe = os.path.join(recipe_root, "class_incr.yaml")
```

Comment:
I think it needs to be declared first, either None or as it is.

Reply:
please take a look